server/routes/api/delete/deleteUserRoutes.ts

```diff
@@ -0,0 +1,75 @@
+import { Router, Request, Response } from "express";
+import { validationResult, param } from "express-validator";
+
+import Handler from "./deleteUserHandler";
+import { END_TIMESTAMP, 
+         BABYSITTER_INVALID_INPUT_ERROR, 
+         PARENT_INVALID_INPUT_ERROR } from "../../../utils/global/globals"
+
+const deleteRouter = Router();
+
+const handler = new Handler();
+
+deleteRouter.put("/parent/:parent", 
+    [ 
+        param('parent').notEmpty().isNumeric().withMessage(PARENT_INVALID_INPUT_ERROR)
+    ],
+    async (req: Request, res: Response) => {
+        try {
+            const fieldValidationResult = validationResult(req);
+            if (!fieldValidationResult.isEmpty()) {
+              return res
+                .status(400)
+                .json({ message: fieldValidationResult.array().map((item) => item.msg).join(' ') });
+            }
+            const { parent: parentId } = req.params
+            const data = await handler.getParent(Number(parentId));
+            const userEndTimestamp = data[0]['end_timestamp']
+
+            if (userEndTimestamp != String(END_TIMESTAMP)) {
```

Comment:
i would put this in the validation function. Error 400 is unvalid because of vallidation